assets/src/blocks/HubspotForm/HubspotFormContext.js

```diff
@@ -0,0 +1,64 @@
+import {
+  createContext,
+  useEffect,
+  useRef,
+  useState,
+} from '@wordpress/element';
+
+const Context = createContext({});
+Context.displayName = 'HubspotFormContext';
+const { Provider, Consumer } = Context;
+
+const ContextProvider = ({
```

Comment:
Moved to a hook. However, I left the Context only for taking and returning props instead of passing them through each children `Component`.

Reply:
As discussed on Slack, we should avoid using Context here for now because there's no benefit to using it, it's used in a single place. Context only has benefits for props that get passed deeply, which shouldn't be the case for a block's attributes.